src/scratch/BlockMenus.as

```diff
@@ -533,9 +533,28 @@ public class BlockMenus implements DragClient {
 		var m:Menu = new Menu(null, 'proc');
 		addGenericBlockItems(m);
 		m.addItem('edit', editProcSpec);
+		if (block.op==Specs.CALL) {
+			/// TLF: alternative text? -maybe "show definition"...?
+			m.addItem('go to definition', gotoProcSpec);
```

Comment:
OK, I made the text change, plus a couple of minor tweaks.
My only comment after trying it out as "see the define script" would be that it makes the menu seem a bit wide compared to others (might some translations make it even wider?), and a tad unusual since other menu items (and tooltips) don't really use articles, i.e. "the", "a", etc...
Shall we start up SUREMI (Society for the Use of Real English in Menu Items)... ;)